deepchecks/__init__.py

```diff
@@ -9,13 +9,23 @@
 # ----------------------------------------------------------------------------
 #
 """Deepchecks."""
-import os
-import pathlib
 import sys
 import types
 import warnings
 from importlib._bootstrap import _init_module_attrs
 
+try:
+    from importlib.metadata import version
+except ImportError:
+    # NOTE:
+    # 'importlib.metadata' was added to python>=3.8
```

Comment:
So maybe we need to add this library to our requirements?

Reply:
I added 
`importlib_metadata>=1.4; python_version < '3.8'`